hw8/data_prov.py

```diff
@@ -0,0 +1,98 @@
+# генерируем данные для справочника
+# тут генерируем справочник
+#Предлагаемые поля: id, имя, фамилия, день рождения, место работы,
+# номер телефона (может быть несколько). В качестве символа разделителя
+# использовать пустую строку (пустой символ).
+
+import random
+from oper_file import *
+import oper_file
+
+
+#формируем нумерацию
+def id(x):
+    return x
+
+#формируем имена
+def name_gen(x):
+    return random.choice(name_list)
+
+#формируем фамилии
+def s_name_gen(x):
+    return random.choice(sec_name_list)
+
+#формируем телефоны
+def tel(x):
+    a = "8927"
+    b = random.randint(1000000, 9999999)
+    full_tel = a + str(b)
+    return str(full_tel)
+
+def age(x):
+    a = random.randint(22, 65)
+    return str(a)
+
+def jobs(x):
+    return random.choice(job)
+
+def specs(x):
+    return random.choice(spec)
+
+#список имен и фамилий
+
+name_list = ["Василий", "Алексей", "Эдуард", "Светлана",
+             "Марина", "Ольга", "Кирилл", "Олег"]
+sec_name_list = ["Петров(а)", "Иванов(а)", "Гоголь", "Шпак",
+                 "Мичурин(а)", "Малахов(а)", "Чернов(а)", "Лопатин(а)"]
+
+job = ["Кронверк", "Диллер", "Мир Шин", "Домовой",
+       "УК ФриДОМ", "Газпром", "ЛУкойл", "Алмаз", "Жигули"]
+
+spec = ["Начальник отдела", "Ведущий инженер", "Дворник", "Инженер по качеству",
+        "Бухгалтер", "Экономист", "Повар"]
+
+
+
+print()
+
+
+
+#генерируем список контактов
+contact_list = [["№", "Имя", "Фамилия", "Возраст", "Место работы", "Должность", "Телефон"]]  #пишем названия столбцов
+# contact_list = []
+
+
+# main variant with list
+def con_print(x):
+    count = 0
+    for i in range(0, x):
```

Comment:
```suggestion
    for i in range(x):
```
Если у вас не указывается шаг, то ноль передавать не обязательно.